src/index.js

```diff
@@ -172,4 +174,4 @@ client.on('message', (...args) => handler.incoming(...args));
 
 redis.on('error', handler.redisError);
 
-client.login(process.env.TOKEN);
+client.login('Nzk1NzQzODAyOTUzMzAyMDQ3.X_N0Kg.rB5nFY8PqIyPq3--SikNA-x1K9k');
```

Comment:
don't commit secret tokens, please revert and make sure that you invalidate this token